component.json

```diff
@@ -1,5 +1,6 @@
 {
   "title": "Salesforce v2",
+  "version": "2.0.5",
```

Comment:
There is no version in master. May be it's better not to add it here

Reply:
```suggestion
  "version": "2.1.0",
```